src/components/AppSlider/index.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import AppSlider, { appSliderDataTestIds } from '.';
+
+describe('AppSlider', () => {
+  it('renders AppSlider component', () => {
+    render(<AppSlider onValueChanged={jest.fn()} />);
+
+    const appSlider = screen.getByTestId(appSliderDataTestIds.base);
+
+    expect(appSlider).toBeVisible();
```

Comment:
Should we also check the `Slider` component inside?

Reply:
@hoangmirs added in [c732b51](https://github.com/manh-t/react-survey/pull/74/commits/c732b515c38639e2c990f10125dcc9ca57df594c)